src/posts/2021-06-14-Everyday_Accessibility.md

```diff
@@ -0,0 +1,121 @@
+	<figcaption>Explicitly select format options like "Numbered List" in Microsoft Word to increase accessibility.</figcaption>
+</figure>
+
+### Include document structure
+Headings (or titles on slides) provide a great way for everyone to preview, skim, and search content. Structure is particularly helpful for blind readers who want to get an overview of the document before reading line-by-line. Include headings throughout a document and titles on every slide, and remember to use the actual “Heading” and “Title” features of the program, not simply stylized text.
+
+<figure role="figure" aria-label="Using PowerPoint's built-in title placeholders is important for accessibility.">
+	<img alt="Two simple PowerPoint slides each have titles. The mouse cursor hovers over the second slide's thumbnail to show the title text appears as mouse hover content when properly formatted." src="/img/posts/2021-06-14-Everyday_Accessibility/PPT_Titles.png" />
+	<figcaption>Using PowerPoint's built-in "title" placeholders is important for accessibility.</figcaption>
+</figure>
+
+### Add alt text to images
+Just like with social media, you can add alt text descriptions to images in documents as well. You will typically see an input box to add descriptions grouped with other options when you insert the image.
+
+<figure role="figure" aria-label="Add image descriptions (alt text) to images in documents, as in Microsoft Word's Alt Text option.">
+	<img alt="Word document has photo inserted and selected. In the right pane, the Alt Text option is open and showing an input field that reads 'White and yellow flowers'." src="/img/posts/2021-06-14-Everyday_Accessibility/Windows-AltText.png" />
+	<figcaption>Add image descriptions ("alt text") to images in documents, as in Microsoft Word's Alt Text option.</figcaption>
+</figure>
+
+### Use accessibility checkers
+Microsoft products (namely Word and PowerPoint) come with a “Check Accessibility” option. Use this to double-check you’ve addressed key accessibility needs.
+
+<figure role="figure" aria-label="Microsoft's Check Accessibility feature helps ensure your document is inclusive.">
+	<img alt="The Check Accessibility menu option is activated on a simple PowerPoint deck where one slide is missing a title. The Accessibility pane shows the 'Missing slide title' error and the mouse hover text for that slide also reads '[No Title]'." src="/img/posts/2021-06-14-Everyday_Accessibility/PPT_A11yChecker.png" />
+	<figcaption>Microsoft's "Check Accessibility" feature helps ensure your document is inclusive.</figcaption>
+</figure>
+
+## Audio and Video Content
+
+### Have clear audio
+Though all of our devices can record audio and video on-demand, sound quality can vary.  Be sure to speak as clearly as possible, preferably at a steady pace and reasonable volume, and remove or avoid background noise including echo, empty room effect, humming, or white noise.
```

Comment:
Can we add a sentence about who or how this helps?

Reply:
Great catch! Will add.